{{cookiecutter.repository_name}}/.globality/build.json

```diff
@@ -1,5 +1,13 @@
 {
   "params": {
+    "entrypoint": {
+      "pre_service_commands": [
+        "pip install awscli",
+        "aws s3 cp ${MODEL_LOCATION} .",
+        "mkdir -p /opt/ml/model",
+        "tar -zxf model.tar.gz -C /opt/ml/model"
+      ]
+    },
```

Comment:
I believe this stuff is old.  I think you can find the latest here: https://github.com/globality-corp/experimental-matcher/blob/master/.globality/build.json.  @pankajghosh anything else that needs updating to latest CD setup?

Reply:
yup, I would use existing AI service build.json as a template for this. Few things:
1. No need for entrypoint section anymore.
2. environment section should only have a "demo" section(since we do not autodeploy to devx)
3. type should be ai-model
4. update_commands needs to be fixed to use v2 (new CD command)